src/staticanalysis/bot/static_analysis_bot/config.py

```diff
@@ -101,6 +102,7 @@ def setup(self,
             self.cov_url = cov_config.get('server_url')
             self.cov_auth = cov_config.get('auth_key')
             self.cov_package_ver = cov_config.get('package_ver')
+            self.cov_full_stack = cov_config.get('full_stack', False)
```

Comment:
Do we want to make it possible to enable/disable it, or do we want it always enabled?

Reply:
Yes, what if for a reason we want to disable this, we will not redeploy just modify a parameter from the secret.

Reply:
OK, I would remove it once we confirm it is working as we expect though, otherwise we make the code too complex (we already have a lot of configuration variables and different branches).

Reply:
Sure we will do that.